napari/layers/_layer_actions.py

```diff
@@ -191,7 +191,16 @@ def _project(ll: LayerList, axis: int = 0, mode: str = 'max') -> None:
     meta = {
         key: layer._get_base_state()[key]
         for key in layer._get_base_state()
-        if key not in ('scale', 'translate', 'rotate', 'shear', 'affine')
```

Comment:
This is actually incorrect, but not the fault of this PR... Should we make an issue? Projection should *at least* subsample the transforms for the given axes. (It's more complicated if there are transformations that are not orthogonal to the projection axis, but that can tackle those separately.)

Reply:
I think a comment would be very useful! 👍

Reply:
is it ok now?